tests/functional/data_masking/test_aws_encryption_sdk.py

```diff
@@ -1,9 +1,7 @@
-from __future__ import annotations
```

Comment:
I've preferred removing the `from __future__ import annotations` from the few test files that had it because it would result in less changes in the PR and also because we're not aiming to have the import in tests for now.

Reply:
Ok ok! Agree!